### PR TITLE
Add grid component and in document too.

### PR DESCRIPTION
**Pull request by rutu14**

### CODE REVIEW

**Comment by Ranjeetk489** [comment]
looks good to go

**Review by uday-nv-work — APPROVED** [review]
Looks good

File: documentation/css/style.css [hunk 1]
```diff
@@ -201,3 +201,22 @@
...
+    border-radius: 0;
+}
+
+.grid-con{
+    width: 70rem;
+    height: 30rem;
+}
+
+.doc-code{
+    background: #f0f5a2;
```
Comment (zoef-yt) [inline documentation/css/style.css:217]: use var for colours

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: documentation/css/style.css (hunk 1 above)
Comment (rutu14) [reply documentation/css/style.css:217]: Sure

**Pull request merged**